pkg/config/yaml.go

```diff
@@ -74,3 +74,31 @@ func FileConfigSpec(fileName string) (*v1alpha1.TracingPolicySpec, error) {
 func FileConfigYaml(fileName string) (*GenericTracingConf, error) {
 	return fileConfig(fileName)
 }
+
+type MetadataNamespaced struct {
+	Name      string `yaml:"name"`
+	Namespace string `yaml:"namespace"`
```

Comment:
Is this namespace field has to be explicitly set in the namespaced tracing policy?  or are we able to populate it from something like `kubectl command -n namespace` ?

Reply:
Both.